sphinx/templates/quickstart/conf.py_t

```diff
@@ -1,13 +1,12 @@
 # -*- coding: utf-8 -*-
 #
-# {{ project }} documentation build configuration file, created by
-# sphinx-quickstart on {{ now }}.
+# Configuration file for the Sphinx documentation builder.
```

Comment:
I think ` for Sphinx` is simple and better. What do you think?

Reply:
I've added the "documentation builder" part to give people unfamiliar with Sphinx a hint to the real purpose of the file (configure the documentation builder). I can also remove it. Either is fine by me.

Reply:
Either is fine to me too. So go as you like it.

I guess the documents are usually placed at `docs` directory or `foobar-docs` repository. So users might infer it's configuration file for documentation tooling (am I optimistic?).

Reply:
In that case, I slightly perfer the way I've originally written it. I'll leave it as is. By me, the PR is ready to be merged.